stock_prices/stock_prices.py

```diff
@@ -3,7 +3,14 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  max_prof = prices[1] - prices[0]
+  for i in range(0, len(prices)):
```

Comment:
looks very clean! however, this solution is O(n**2). After you're done with MVP, see if you can bring it down to O(n)